md2html/md2html.c

```diff
@@ -288,7 +294,7 @@ static const char* input_path = NULL;
 static const char* output_path = NULL;
 
 static int
-cmdline_callback(int opt, char const* value, void* data)
+cmdline_callback(int opt, char const* value, MD_UNUSED void* data)
```

Comment:
I wasn't sure if this was a form of "documentation" so rather than removing this unused argument I decorated it so that the compiler knows its intentionally unused.